### PR TITLE
Tidy up UI node docs

### PR DESCRIPTION
**Pull request by rparrett**
# Objective

While reviewing #10187 I noticed some other mistakes in the UI node docs.

## Solution

I did a quick proofreading pass and fixed a few things. And of course, the typo from that other PR.

## Notes

I occasionally insert a period to make a section of doc self-consistent but didn't go one way or the other on all periods in the file.

### CODE REVIEW

**Review by alice-i-cecile — APPROVED** [review]

File: crates/bevy_ui/src/ui_node.rs [hunk 1]
```diff
@@ -944,7 +951,7 @@ impl GridTrack {
 
     /// Create a grid track with an `fr` size.
     /// Note that this will give the track a content-based minimum size.
-    /// Usually you are best off using `GridTrack::flex` instead which uses a zero minimum size
+    /// Usually you are best off using `GridTrack::flex` instead which uses a zero minimum size.
     pub fn fr<T: From<Self>>(value: f32) -> T {
         Self {
             min_sizing_function: MinTrackSizingFunction::Auto,
```
Comment (NoahShomette) [inline crates/bevy_ui/src/ui_node.rs:957]: LINE 963

It should be "with *a* `minmax(...`

File: crates/bevy_ui/src/ui_node.rs [hunk 2]
```diff
@@ -1116,7 +1123,7 @@ impl RepeatedGridTrack {
 
     /// Create a repeating set of grid tracks with an `fr` size.
     /// Note that this will give the track a content-based minimum size.
-    /// Usually you are best off using `GridTrack::flex` instead which uses a zero minimum size
+    /// Usually you are best off using `GridTrack::flex` instead which uses a zero minimum size.
     pub fn fr<T: From<Self>>(repetition: u16, value: f32) -> T {
         Self {
             repetition: GridTrackRepetition::Count(repetition),
```
Comment (NoahShomette) [inline crates/bevy_ui/src/ui_node.rs:1129]: LINE 1135

Should be "with *a* `minmax...`

File: crates/bevy_ui/src/ui_node.rs [hunk 3]
```diff
@@ -1240,16 +1247,23 @@ impl From<RepeatedGridTrack> for Vec<RepeatedGridTrack> {
 ///
 /// The default `span` is 1. If neither `start` or `end` is set then the item will be placed automatically.
 ///
-/// Generally, at most two fields should be set. If all three fields are specified then `span` will be ignored. If `end` specifies an earlier
+/// Generally, at most, two fields should be set. If all three fields are specified then `span` will be ignored. If `end` specifies an earlier
```
Comment (NoahShomette) [inline crates/bevy_ui/src/ui_node.rs:1250]: I dont believe that the added comma should be there, it flows better and makes more intuitive sense without it or slightly reworded.
```suggestion
/// Generally, two fields should be set at most. If all three fields are specified then `span` will be ignored. If `end` specifies an earlier
```

File: crates/bevy_ui/src/ui_node.rs [hunk 4]
```diff
@@ -1607,13 +1621,13 @@ pub struct CalculatedClip {
 ///
 /// UI nodes that have the same z-index will appear according to the order in which they
 /// appear in the UI hierarchy. In such a case, the last node to be added to its parent
-/// will appear in front of this parent's other children.
+/// will appear in front its siblings.
```
Comment (NoahShomette) [inline crates/bevy_ui/src/ui_node.rs:1624]:
```suggestion
/// will appear in front of its siblings.
```

**Review by NoahShomette — APPROVED** [review]
Several of these are nitpicks, I don't have strong opinions on the clear nitpicks but as long as the documentation of this file is the focus might as well fix the smaller 'errors' while we are here imo. Approving just cuz most of it is small but there are some genuine errors in there are as well

File: crates/bevy_ui/src/ui_node.rs [hunk 5]
```diff
@@ -639,7 +646,7 @@ impl Default for JustifyContent {
 
 /// Defines the text direction
 ///
-/// For example English is written LTR (left-to-right) while Arabic is written RTL (right-to-left).
+/// For example, English is written LTR (left-to-right) while Arabic is written RTL (right-to-left).
 #[derive(Copy, Clone, PartialEq, Eq, Debug, Serialize, Deserialize, Reflect)]
 #[reflect(PartialEq, Serialize, Deserialize)]
 pub enum Direction {
```
[the remark this answers is not in the document]
Comment (rparrett) [reply crates/bevy_ui/src/ui_node.rs:652]: I don't really want to rewrite all of the docs in this PR, but I like your suggestion for 671.

**Comment by rparrett** [comment]
I think I got all of your feedback incorporated. Thanks!

File: crates/bevy_ui/src/ui_node.rs [hunk 6]
```diff
@@ -583,9 +590,9 @@ pub enum AlignContent {
     /// Each line will stretch to fill the remaining space.
     Stretch,
     /// Each line fills the space it needs, putting the remaining space, if any
-    /// inbetween the lines.
+    /// between the lines.
     SpaceBetween,
-    /// The gap between the first and last items is exactly THE SAME as the gap between items.
+    /// The gap between the first and last items is exactly the same as the gap between items.
```
[the remark this answers is not in the document]
Comment (rparrett) [reply crates/bevy_ui/src/ui_node.rs:595]: I played around with this a bit, but I would really want to adjust the language of all the spacing options for consistency. Maybe we could save that for a follow up?

[the remark this answers is not in the document]
Comment (CrumbsTrace) [reply crates/bevy_ui/src/ui_node.rs:595]: Sounds good. They're pretty inconsistent with the terms they use (line vs item) and don't really follow pattern in how they're phrased. Might be better for a follow up PR

[the remark this answers is not in the document]
Comment (rparrett) [reply crates/bevy_ui/src/ui_node.rs:595]: Filed here so we don't forget: https://github.com/bevyengine/bevy/issues/10218

**Pull request merged**